src/js/node/assert.ts

```diff
@@ -1,1257 +1,1016 @@
-    }
-    function _setPrototypeOf(o, p) {
-      return (
-        (_setPrototypeOf =
-          Object.setPrototypeOf ||
-          function (o2, p2) {
-            return (o2.__proto__ = p2), o2;
-          }),
-        _setPrototypeOf(o, p)
-      );
-    }
-    var codes = {},
-      assert,
-      util;
-    function createErrorType(code, message, Base) {
-      Base || (Base = Error);
-      function getMessage(arg1, arg2, arg3) {
-        return typeof message == "string" ? message : message(arg1, arg2, arg3);
-      }
-      var NodeError = /* @__PURE__ */ (function (_Base) {
-        _inherits(NodeError2, _Base);
-        function NodeError2(arg1, arg2, arg3) {
-          var _this;
-          return (
-            _classCallCheck(this, NodeError2),
-            (_this = _possibleConstructorReturn(
-              this,
-              _getPrototypeOf(NodeError2).$call(this, getMessage(arg1, arg2, arg3)),
-            )),
-            (_this.code = code),
-            _this
-          );
-        }
-        return NodeError2;
-      })(Base);
-      codes[code] = NodeError;
-    }
-    function oneOf(expected, thing) {
-      if (Array.isArray(expected)) {
-        var len = expected.length;
-        return (
-          (expected = expected.map(function (i) {
-            return String(i);
-          })),
-          len > 2
-            ? "one of ".concat(thing, " ").concat(expected.slice(0, len - 1).join(", "), ", or ") + expected[len - 1]
-            : len === 2
-              ? "one of ".concat(thing, " ").concat(expected[0], " or ").concat(expected[1])
-              : "of ".concat(thing, " ").concat(expected[0])
-        );
-      } else return "of ".concat(thing, " ").concat(String(expected));
-    }
-    function startsWith(str, search, pos) {
-      return str.substr(!pos || pos < 0 ? 0 : +pos, search.length) === search;
-    }
-    function endsWith(str, search, this_len) {
-      return (
-        (this_len === void 0 || this_len > str.length) && (this_len = str.length),
-        str.substring(this_len - search.length, this_len) === search
-      );
-    }
-    function includes(str, search, start) {
-      return (
-        typeof start != "number" && (start = 0),
-        start + search.length > str.length ? !1 : str.indexOf(search, start) !== -1
+const {
+  ArrayFrom,
+  ArrayIsArray,
+  ArrayPrototypeIndexOf,
+  ArrayPrototypeJoin,
+  ArrayPrototypePush,
+  ArrayPrototypeSlice,
+  Error,
+  FunctionPrototypeCall,
+  MapPrototypeDelete,
+  MapPrototypeGet,
+  MapPrototypeHas,
+  MapPrototypeSet,
+  NumberIsNaN,
+  ObjectAssign,
+  ObjectIs,
+  ObjectKeys,
+  ObjectPrototypeIsPrototypeOf,
+  ReflectApply,
+  ReflectHas,
+  ReflectOwnKeys,
+  RegExpPrototypeExec,
+  SafeMap,
+  SafeSet,
+  SafeWeakSet,
+  StringPrototypeIndexOf,
+  StringPrototypeSlice,
+  StringPrototypeSplit,
+  SymbolIterator,
+} = require("internal/primordials");
+
+const { Buffer } = require("node:buffer");
+const { deprecate } = require("node:util");
```

Comment:
did we need to import node:util at the top of this file (i.e. in all cases)?

Reply:
Refactored to be lazy loaded